docs/content/en/docs/Getting started/_index.md

```diff
@@ -28,7 +30,7 @@ Once the installation is complete, you can begin using your Kubernetes cluster.
 ## Download
 
 1. Visit the Kairos [release page on GitHub](https://github.com/kairos-io/provider-kairos/releases)
-1. Select the latest release and download the assets of your flavor. For example, pick the [kairos-opensuse-v1.0.0-k3sv1.24.3+k3s1.iso](https://github.com/kairos-io/provider-kairos/releases/download/v1.0.0/kairos-opensuse-v1.0.0-k3sv1.24.3+k3s1.iso) ISO file for the openSUSE based version, where `v1.24.3+k3s1` in the name is the `k3s` version and `v1.0.0` is the Kairos one to deploy on a VM.
+1. Select the latest release and download the assets of your flavor. For example, pick the [kairos-opensuse-v{{< kairosVersion >}}-k3sv1.24.3+k3s1.iso](https://github.com/kairos-io/provider-kairos/releases/download/v/kairos-opensuse-v{{< kairosVersion >}}-k3sv1.24.3+k3s1.iso) ISO file for the openSUSE based version, where `v1.24.3+k3s1` in the name is the `k3s` version and `v{{< kairosVersion >}}` is the Kairos one to deploy on a VM.
```

Comment:
I think this will be problematic as soon as the k3s version changes

Reply:
Yes. You are correct. I would like to make all the numbers use data from short code.

Reply:
I added `k3sVersion` as a shortcode, ofc we can add more!